### ISSUE TITLE
Not compatible with Python 3.13

### ISSUE THREAD

**Issue by dertuxmalwieder** [beetbox/pyacoustid#87]
Python 3.13 removed the `sunau` module, so pyacoustid won't work.

**Comment by byemc** [comment]
Python 3.13 also removes `aifc`.

**Comment by dertuxmalwieder** [comment]
Indeed, both are broken. But I wasnt sure which depends on what. Thank you.

**Comment by byemc** [comment]
It does look like `aifc` not being included is actually an issue with a dependency by the same author, `audioread`. On [an issue on *that* issue tracker](https://github.com/beetbox/audioread/issues/144) I found someone published the dead modules on PyPi: https://github.com/youknowone/python-deadlib

I got it working by installing `standard-aifc` and `standard-sunau`.

**Comment by semohr** [comment]
Since `audioread` now supports Python 3.13, I’m going to close this issue. If you encounter any further problems, please feel free to reopen it!
